### PR TITLE
Append content of contributing.md to PULLREQ_EDITMSG if it exists

### PR DESCRIPTION
**Pull request by hatricker**
per discussion with @alexgerstein

### CODE REVIEW

File: lib/hub/commands.rb [hunk 1]
```diff
@@ -243,11 +243,16 @@ def pull_request(args)
...
-          msg.puts "#{cc} Requesting a pull to #{base_project.owner}:#{options[:base]} from #{options[:head]}"
-          msg.puts "#{cc}"
-          msg.puts "#{cc} Write a message for this pull request. The first block"
-          msg.puts "#{cc} of text is the title and the rest is description."
+
+          if contributing_md_file
+            msg.puts File.readlines(contributing_md_file).map {|line| line.prepend('# ')}.join
+          else
+            msg.puts ""
+            msg.puts "#{cc} Requesting a pull to #{base_project.owner}:#{options[:base]} from #{options[:head]}"
```
Comment (alexgerstein) [inline lib/hub/commands.rb]: Do we think there's value in keeping these notes around?

Comment (hatricker) [inline lib/hub/commands.rb]: Yes, hub is shared by all Gusto repos, right?

Comment (alexgerstein) [inline lib/hub/commands.rb]: I think so. I mean should we append this text, even if we have a contributing.md?

Comment (hatricker) [inline lib/hub/commands.rb]: we probably can keep ```"#{cc} Requesting a pull to #{base_project.owner}:#{options[:base]} from #{options[:head]}"```  and move the rest to contributing.md

**Comment by alexgerstein** [comment]
@zp-alfred Accept!

**Comment by zp-alfred** [comment]
Accepted by alexgerstein!

**Pull request merged**